fdbclient/Schemas.cpp

```diff
@@ -143,6 +143,7 @@ const KeyRef JSONSchemas::statusSchema = R"statusSchema(
                         "coordinator"
                      ]
                   },
+				  "tss":false,
```

Comment:
Nit: this formatting seems extra indented? Do we ignore clang-format for this file?